x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/ExplainLifecycleRequest.java

```diff
@@ -21,13 +23,48 @@
  * {@link #indices(String...)} method
  */
 public class ExplainLifecycleRequest extends ClusterInfoRequest<ExplainLifecycleRequest> {
+    private static final Version FILTERS_INTRODUCED_VERSION = Version.V_8_0_0;
```

Comment:
TODO: Update this in the backport and in master after merging the backport.

Reply:
Updated the serialization version in https://github.com/elastic/elasticsearch/commit/a76242d43d1a668b45be6c48f24a5dfde464038e now that the backport has been merged.